app/views/components/returns/checkYourAnswer/credits/NoCreditsClaimed.scala.html

```diff
@@ -0,0 +1,34 @@
+@*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import components.{Link, paragraph}
+@this(
+    paragraph: paragraph,
+    link: Link
+)
+
+@()(implicit request: Request[_], messages: Messages)
+
+@paragraph(
+    content = Html(messages("submit-return.check-your-answers.credits.not.claimed.hint"))
+)
+
+<!-- todo: add a proper link for the credit change -->
```

Comment:
is this on the work flowy todo list?

Reply:
add it thanks

Reply:
sorry, "have added it thanks"... can't type this morning